_includes/head.html

```diff
@@ -7,6 +7,11 @@
     <!-- TODO: configure SEO/OG tags -->
     <meta name="description" content="{% if page.excerpt %}{{ page.excerpt | strip_html | strip_newlines | truncate: 160 }}{% else %}{{ site.description }}{% endif %}">
 
+    <!-- Bootstrap 4 includes -->
```

Comment:
ideally we'd only include these scripts on pages that need it -- can we condition this on the page calling the include?

Reply:
Yes I added it in the next commit